test/Bin/GameTest/GameTests.cpp

```diff
@@ -1705,3 +1707,63 @@ GAME_TEST(Issues, Issue1331) {
     auto damageRange = hpsTape.reversed().adjacentDeltas().flattened().filtered([] (int damage) { return damage > 0; }).minMax();
     EXPECT_EQ(damageRange, tape(1 * 2, (43 + 13) * 2));
 }
+
+GAME_TEST(Issues, Issue1338) {
+    // Casting telepathy on an actor and then killing it results in the actor not dropping any gold.
+    auto deadTape = actorTapes.indicesByState(AIState::Dead);
+    auto statusTape = tapes.statusBar();
+    auto goldTape = tapes.gold();
+    auto peasantGoldTape = tapes.custom([] { return pActors[18].items[3].goldAmount; });
+    test.playTraceFromTestData("issue_1338.mm7", "issue_1338.json");
+    EXPECT_EQ(deadTape, tape(std::initializer_list<int>{}, {18}, std::initializer_list<int>{})); // Alive -> Dead -> corpse picked up.
+    EXPECT_GT(peasantGoldTape.max(), 0); // Peasant should have had gold generated.
+    EXPECT_EQ(goldTape.delta(), peasantGoldTape.max());
+    EXPECT_TRUE(statusTape.contains(fmt::format("{} gold", peasantGoldTape.max()))); // Telepathy status message.
+    EXPECT_TRUE(statusTape.contains(fmt::format("You found {} gold!", peasantGoldTape.max()))); // Corpse pickup message.
```

Comment:
I understand that tests will likely use english localization. But still...
Maybe tape party gold and ensure that it have 3 entries (initial, after telekinesis, after corpse pickup)?

Reply:
Unfortunately, this is currently the only way to check that telepathy was cast. The only observable effect of telepathy that can be taped with the current machinery is a status message of "X gold" (telepathy doesn't change party gold, it only displays how much gold the monster carries).

So, I made peace with the fact that we're testing localizable messages. For now, at least.

Reply:
Ah yes. Seems I mistook telepathy for telekinesis.
Well okay then. Status it is.